### PR TITLE
Add test config for resnetrs

### PR DESCRIPTION
**Pull request by fyangf**
GPU test disabled for now due to EMA incompatible on GPU.

### CODE REVIEW

**Comment by will-cromar** [comment]
Can you fix this formatting?

```
--- a/tests/tensorflow/nightly/classifier-resnetrs.libsonnet
+++ b/tests/tensorflow/nightly/classifier-resnetrs.libsonnet
@@ -87,7 +87,7 @@ local tpus = import 'templates/tpus.libsonnet';
       task+: {
         train_data+: {
           global_batch_size: 128,
-         },
+        },
         validation_data+: {
           global_batch_size: 128,
         },
```

LGTM otherwise

File: tests/tensorflow/nightly/classifier-resnetrs.libsonnet [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+    },
+  },
+  local k80 = gpu_common {
+    local config = self,
+
+    paramsOverride+:: {
+      task+: {
+        train_data+: {
+          global_batch_size: 128,
+         },
```
Comment (allenwang28) [inline tests/tensorflow/nightly/classifier-resnetrs.libsonnet:90]: Could you remove one space here to fix the spacing? I believe this is breaking the presubmit checks

Comment (fyangf) [reply tests/tensorflow/nightly/classifier-resnetrs.libsonnet]: Removed whitespace.

**Review by allenwang28 — APPROVED** [review]

**Review by will-cromar — APPROVED** [review]

**Comment by allenwang28** [comment]
Thanks Fan!

**Pull request merged**